### PR TITLE
Add ADR for production docker setup

### PR DESCRIPTION
**Pull request by rogerhutchings**
Adds an ADR for the current production build setup, and updates the README.

### CODE REVIEW

**Comment by rogerhutchings** [comment]
cc #560

File: README.md [hunk 1]
```diff
@@ -6,9 +6,25 @@
...
+  - [NPM](#npm)
+  - [Packages directory](#packages-directory)
+- [Production deployment](#production-deployment)
+- [License](#license)
+
 ## Requirements
 
-[Browser support](docs/arch/adr-3.md)
+- [Browser support](docs/arch/adr-3.md)
+- Node 8
```
Comment (srallen) [inline README.md]: Node 10 is the LTS now. Maybe this should be >=8?

Comment (rogerhutchings) [reply README.md]: Sure, but we're not using it anywhere; I'm happy to do a separate PR to bump to 10 across the repo.

Comment (srallen) [reply README.md]: I'm ok with separate update PR. Thanks.

File: bin/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+1. Install the top-level dependencies for the repo
+1. Install the remaining dependencies for each package, and symlink them together
+1. Build `@zooniverse/async-states`
+1. Build `@zooniverse/grommet-theme`
+1. Build `@zooniverse/react-components`
+1. Build `@zooniverse/lib-classifier`
+
+## `get-app-folders.sh`
+
+Returns a comma-separate list of absolute paths to apps in the package folder, for use in build scripts.
```
Comment (adammcmaster) [inline bin/README.md]: "comma-separated"

File: README.md (hunk 1 above)
Comment (rogerhutchings) [reply README.md]: cc #595

**Comment by rogerhutchings** [comment]
Sweet - I've fixed that typo, so I'm going to go ahead and merge

**Comment by rogerhutchings** [comment]
cc #156

**Pull request merged**